swerve100/src/main/java/org/team100/lib/motor/turning/CANTurningMotor.java

```diff
@@ -66,12 +61,19 @@ public void set(double output) {
         m_motor.set(output);
     }
 
-    public void setPID(ControlMode control, double output) {
-        int rotationsPerRev = 1666;
-        double goal = m_motor.getSelectedSensorPosition()
-                + MathUtil.angleModulus(2 * Math.PI * (output - m_motor.getSelectedSensorPosition() / rotationsPerRev))
-                        / (2 * Math.PI) * rotationsPerRev;
-        m_motor.set(control, goal);
+    public void setPIDVelocity(double outputRadiansPerSec) {
+        double ticksPerRevolution = 1024;
```

Comment:
since there's a lot of duplicated logic between the TalonFX and TalonSRX versions of this file, maybe extract it to a "util" class of some kind?